docs/content/preview/api/ysql/the-sql-language/statements/txn_begin.md

```diff
@@ -17,7 +17,11 @@ Use the `BEGIN` statement to start a transaction with the default (or specified)
 ## Syntax
 
 {{%ebnf%}}
-  begin
+  begin,
+  transaction_mode,
+  isolation_level,
+  read_write_mode,
+  deferrable_mode
```

Comment:
We need to add an example with this. The current example is  only 
`BEGIN TRANSACTION; SET TRANSACTION ISOLATION LEVEL REPEATABLE READ;`